bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rntuple.py

```diff
@@ -93,10 +113,33 @@ def _RNTupleReader_LoadEntry(self, *args):
 
 @pythonization("RNTupleReader", ns="ROOT::Experimental")
 def pythonize_RNTupleReader(klass):
+    klass._Open = klass.Open
+    klass.Open = _RNTupleReader_Open
+
     klass._LoadEntry = klass.LoadEntry
     klass.LoadEntry = _RNTupleReader_LoadEntry
 
 
+def _RNTupleWriter_Append(model, *args):
+    # In Python, the user cannot create REntries directly from a model, so we can safely clone it and avoid destructively passing the user argument.
+    model = model.Clone()
```

Comment:
Maybe we want to have the same check as in `_RNTupleWriter_Recreate` here?

Reply:
We could, but for `RNTupleWriter::Append` the first `model` parameter is mandatory. For `Recreate`, the user can also pass a list of tuples (type, fieldname)`.